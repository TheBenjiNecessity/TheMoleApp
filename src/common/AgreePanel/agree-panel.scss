@import "../../styles/colors";

.player-view {
	display: block;

	height: 1em;
	padding: 0.5rem;
	margin: 0.5rem 0;

	border: 1px solid #138400;
	text-align: center;
	text-overflow: ellipsis;
	overflow: hidden;
}
