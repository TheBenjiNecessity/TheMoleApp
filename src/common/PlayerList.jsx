import React from 'react';
import PlayerListView from './PlayerListView';

const PlayerList = ({ players, animate, inline, xl, lg, md, sm, xs }) => {
	if (!players || !players.length) {
		return null;
	}

	return players.map((player, i) => {
		if (inline) {
			return (
				<div key={i} className="inline">
					<PlayerListView animate={animate} player={player} />
				</div>
			);
		}

		let className = '';

		if (xl) {
			className += ` col-xl-${xl}`;
		}

		if (lg) {
			className += ` col-lg-${lg}`;
		}

		if (md) {
			className += ` col-md-${md}`;
		}

		if (sm) {
			className += ` col-sm-${sm}`;
		}

		if (xs) {
			className += ` col-xs-${xs}`;
		}

		return (
			<div key={i} className={className}>
				<PlayerListView animate={animate} player={player} />
			</div>
		);
	});
};

export default PlayerList;
