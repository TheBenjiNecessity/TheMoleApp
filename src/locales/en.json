{
	"common": {
		"or": "Or",
		"ready_players": "Ready Players:"
	},
	"lobby": {
		"roomcode": "Room Code:",
		"player": "Player:",
		"player_plural": "Players:"
	},
	"welcome": {},
	"mole_reveal_view": {
		"title": "Who is the mole?",
		"first_paragraph": "Look at your devices now to see whether or not you have been selected to be the mole.",
		"second_paragraph": "Warning: do not show the other players your role."
	},
	"episode_start": {
		"title": "Episode {{number}}",
		"first_episode_description": "In a moment, each player's name will be input into the input box one at a time. If you see a green screen after your name has been entered you are safe. If, however, you see a red screen after your name has been entered then you have been eliminated from the game."
	},
	"execution_view": {
		"first_paragraph": 
	},
	"execution_wrapup": {
		"title": "Wrap up",
		"executed_player_0": "We're sad to see {{playerName}} go. They will be missed."
	},
	"challenge_intermission": {
		"generic_text": "generic_text"
	},
	"pre_quiz_intermission": {
		"title": "Pre Quiz",
		"first_paragraph": "In a moment, you will take the quiz; ten questions about the identity of the mole. The questions will may be anywhere from \"How tall is the mole?\" to \"What the mole did during a challenge?\".",
		"second_paragraph": "The player who gets the fewest questions correct will be eliminated from the game."
	},
	"quiz": {
		"title": "Quiz",
		"first_paragraph": "Look at your devices now to complete the quiz.",
		"players_done": "Players finished:"
	},
	"post_quiz_intermission": {
		"title": "Post Quiz",
		"first_paragraph": "Everyone has finished their quizes. Let's see who has been eliminated."
	}
}
