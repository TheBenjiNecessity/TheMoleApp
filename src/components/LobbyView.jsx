import React from 'react';
//import { useTranslation } from 'react-i18next';
import AgreePanel from '../common/AgreePanel';
import NextPanel from '../common/NextPanel';

import PlayerListView from '../common/PlayerListView';

const LobbyView = ({ room, onNext }) => {
	return (
		<NextPanel titleText="Lobby" nextButtonText="Start" onNext={onNext}>
			<div className="form-group pl-xs-0 pr-xs-0 mt-0 col-sm-6">
				<label>Room Code:</label>
				<div className="room-code">{room ? room.roomcode : 'No Code'}</div>
			</div>
			<div className="form-group pl-xs-0 pr-xs-0 mt-0 col-sm-6">
				<label>Players:</label>
				<div className="player-list">
					{room && room.players && room.players.length ? (
						room.players.map((player) => <PlayerListView key={player.name} player={player} />)
					) : (
						<div />
					)}
				</div>
			</div>
			<AgreePanel room={room} />
		</NextPanel>
	);
};

export default LobbyView;
