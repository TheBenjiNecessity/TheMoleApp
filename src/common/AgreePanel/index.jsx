import React, { Fragment } from 'react';
import { useTranslation } from 'react-i18next';

import PlayerList from '../PlayerList';

const AgreePanel = ({ room, inline, xl, lg, md, sm, xs }) => {
	const { t } = useTranslation('common');

	if (!room) {
		return null;
	}

	return (
		<Fragment>
			<p>{t('ready_players')}</p>
			<PlayerList players={room.agreedPlayers} inline={inline} xl={xl} lg={lg} md={md} sm={sm} xs={xs} />
		</Fragment>
	);
};

export default AgreePanel;
