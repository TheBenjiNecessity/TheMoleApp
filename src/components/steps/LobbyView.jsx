import React from 'react';
import { useTranslation } from 'react-i18next';
import AgreePanel from '../../common/AgreePanel';
import PlayerList from '../../common/PlayerList';

const LobbyView = ({ room }) => {
	const { t } = useTranslation('lobby');

	return (
		<div className="panel centered-panel centered-panel-medium next-panel">
			<div className="form-group pl-xs-0 pr-xs-0 mt-0 col-sm-6">
				<label data-testid="roomcodeLabel">{t('roomcode')}</label>
				<div className="room-code" data-testid="roomcode">
					{room ? room.roomcode : 'No Code'}
				</div>
			</div>
			<div className="form-group pl-xs-0 pr-xs-0 mt-0 col-sm-6">
				<label data-testid="playerListLabel">{t('player_plural')}</label>
				<PlayerList players={room.players} xs={12} data-testid="playerList" />
			</div>
			<AgreePanel room={room} inline data-testid="agreePanel" />
		</div>
	);
};

export default LobbyView;
