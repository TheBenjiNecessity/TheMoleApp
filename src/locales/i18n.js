import i18n from 'i18next';
import { initReactI18next } from 'react-i18next';

import en from '../locales/en.json';

i18n
	.use(initReactI18next) // passes i18n down to react-i18next
	.init({
		resources: { en },
		ns: 'common',
		defaultNS: 'common',
		fallbackLng: 'en',
		supportedLngs: [ 'en' ],
		interpolation: {
			escapeValue: false
		}
	});

export default i18n;
