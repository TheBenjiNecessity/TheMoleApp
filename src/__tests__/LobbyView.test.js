import React from 'react';
import { mount } from 'enzyme';
import '../locales/i18n';
import LobbyView from '../components/steps/LobbyView';
import roomSample from './samples/room.sample';
import PlayerList from '../common/PlayerList';
import AgreePanel from '../common/AgreePanel';

describe('MyComponent', () => {
	let room = {};

	beforeEach(() => {
		room = roomSample.sampleRoom(10, 'lobby');
	});

	it('should render lobby correctly', () => {
		const component = mount(<LobbyView room={room} />);

		expect(component).toMatchSnapshot();
	});

	it('should render all lobby components correctly', () => {
		const component = mount(<LobbyView room={room} />);

		expect(component.find('[data-testid="title-text"]').contains('title')).toBe(true);
		expect(component.find('[data-testid="roomcode-label"]').contains('roomcode')).toBe(true);
		expect(component.find('[data-testid="roomcode"]').contains('TEST')).toBe(true);
		expect(component.find('[data-testid="player-list-label"]').contains('player_plural')).toBe(true);
		expect(component.find('[data-testid="player-list"]').type()).toBe(PlayerList);
		expect(component.find('[data-testid="agree-panel"]').type()).toBe(AgreePanel);
	});

	it('should render no roomcode correctly', () => {
		room.roomcode = null;
		const component = mount(<LobbyView room={room} />);

		expect(component.find('[data-testid="roomcode"]').contains('common:no-code')).toBe(true);
	});
});
