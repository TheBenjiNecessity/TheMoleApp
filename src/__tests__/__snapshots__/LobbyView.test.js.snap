// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LobbyView should render lobby correctly 1`] = `
<LobbyView
  room={
    Room {
      "agreedPlayers": Array [],
      "episodes": Array [],
      "players": Array [
        Object {
          "name": "test1",
        },
        Object {
          "name": "test2",
        },
        Object {
          "name": "test3",
        },
        Object {
          "name": "test4",
        },
        Object {
          "name": "test5",
        },
        Object {
          "name": "test6",
        },
        Object {
          "name": "test7",
        },
        Object {
          "name": "test8",
        },
        Object {
          "name": "test9",
        },
        Object {
          "name": "test10",
        },
      ],
      "raisedHands": Object {},
      "roomcode": "TEST",
      "state": "lobby",
    }
  }
>
  <div
    className="panel centered-panel centered-panel-medium next-panel"
  >
    <TopBarTitle
      hasLogo={true}
      title="title"
    >
      <Portal
        containerInfo={
          <div>
            <div
              class="title"
            >
              <span
                class="title-text"
                data-testid="title-text"
              >
                title
              </span>
              <div
                class="logo"
              >
                <div
                  class="title"
                >
                  .
                  <div
                    class="the"
                    style="font-size: 36px;"
                  >
                    The
                  </div>
                  <div
                    class="mole"
                    style="font-size: 45px;"
                  >
                    Mole
                  </div>
                </div>
                <div
                  class="svg"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root"
                    focusable="false"
                    style="font-size: 200px;"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M17.81 4.47c-.08 0-.16-.02-.23-.06C15.66 3.42 14 3 12.01 3c-1.98 0-3.86.47-5.57 1.41-.24.13-.54.04-.68-.2-.13-.24-.04-.55.2-.68C7.82 2.52 9.86 2 12.01 2c2.13 0 3.99.47 6.03 1.52.25.13.34.43.21.67-.09.18-.26.28-.44.28zM3.5 9.72c-.1 0-.2-.03-.29-.09-.23-.16-.28-.47-.12-.7.99-1.4 2.25-2.5 3.75-3.27C9.98 4.04 14 4.03 17.15 5.65c1.5.77 2.76 1.86 3.75 3.25.16.22.11.54-.12.7-.23.16-.54.11-.7-.12-.9-1.26-2.04-2.25-3.39-2.94-2.87-1.47-6.54-1.47-9.4.01-1.36.7-2.5 1.7-3.4 2.96-.08.14-.23.21-.39.21zm6.25 12.07c-.13 0-.26-.05-.35-.15-.87-.87-1.34-1.43-2.01-2.64-.69-1.23-1.05-2.73-1.05-4.34 0-2.97 2.54-5.39 5.66-5.39s5.66 2.42 5.66 5.39c0 .28-.22.5-.5.5s-.5-.22-.5-.5c0-2.42-2.09-4.39-4.66-4.39-2.57 0-4.66 1.97-4.66 4.39 0 1.44.32 2.77.93 3.85.64 1.15 1.08 1.64 1.85 2.42.19.2.19.51 0 .71-.11.1-.24.15-.37.15zm7.17-1.85c-1.19 0-2.24-.3-3.1-.89-1.49-1.01-2.38-2.65-2.38-4.39 0-.28.22-.5.5-.5s.5.22.5.5c0 1.41.72 2.74 1.94 3.56.71.48 1.54.71 2.54.71.24 0 .64-.03 1.04-.1.27-.05.53.13.58.41.05.27-.13.53-.41.58-.57.11-1.07.12-1.21.12zM14.91 22c-.04 0-.09-.01-.13-.02-1.59-.44-2.63-1.03-3.72-2.1-1.4-1.39-2.17-3.24-2.17-5.22 0-1.62 1.38-2.94 3.08-2.94 1.7 0 3.08 1.32 3.08 2.94 0 1.07.93 1.94 2.08 1.94s2.08-.87 2.08-1.94c0-3.77-3.25-6.83-7.25-6.83-2.84 0-5.44 1.58-6.61 4.03-.39.81-.59 1.76-.59 2.8 0 .78.07 2.01.67 3.61.1.26-.03.55-.29.64-.26.1-.55-.04-.64-.29-.49-1.31-.73-2.61-.73-3.96 0-1.2.23-2.29.68-3.24 1.33-2.79 4.28-4.6 7.51-4.6 4.55 0 8.25 3.51 8.25 7.83 0 1.62-1.38 2.94-3.08 2.94s-3.08-1.32-3.08-2.94c0-1.07-.93-1.94-2.08-1.94s-2.08.87-2.08 1.94c0 1.71.66 3.31 1.87 4.51.95.94 1.86 1.46 3.27 1.85.27.07.42.35.35.61-.05.23-.26.38-.47.38z"
                    />
                  </svg>
                </div>
              </div>
            </div>
          </div>
        }
      >
        <div
          className="title"
        >
          <span
            className="title-text"
            data-testid="title-text"
          >
            title
          </span>
          <Logo
            size={200}
            textSize={45}
          >
            <div
              className="logo"
            >
              <div
                className="title"
                style={
                  Object {
                    "color": undefined,
                  }
                }
              >
                <Trans
                  i18nKey="title_logo"
                >
                  .
                  <div
                    className="the"
                    key="1"
                    style={
                      Object {
                        "fontSize": "36px",
                      }
                    }
                  >
                    The
                  </div>
                  <div
                    className="mole"
                    key="2"
                    style={
                      Object {
                        "fontSize": "45px",
                      }
                    }
                  >
                    Mole
                  </div>
                </Trans>
              </div>
              <div
                className="svg"
                style={
                  Object {
                    "color": undefined,
                  }
                }
              >
                <ForwardRef(FingerprintIcon)
                  style={
                    Object {
                      "fontSize": 200,
                    }
                  }
                >
                  <WithStyles(ForwardRef(SvgIcon))
                    style={
                      Object {
                        "fontSize": 200,
                      }
                    }
                  >
                    <ForwardRef(SvgIcon)
                      classes={
                        Object {
                          "colorAction": "MuiSvgIcon-colorAction",
                          "colorDisabled": "MuiSvgIcon-colorDisabled",
                          "colorError": "MuiSvgIcon-colorError",
                          "colorPrimary": "MuiSvgIcon-colorPrimary",
                          "colorSecondary": "MuiSvgIcon-colorSecondary",
                          "fontSizeInherit": "MuiSvgIcon-fontSizeInherit",
                          "fontSizeLarge": "MuiSvgIcon-fontSizeLarge",
                          "fontSizeSmall": "MuiSvgIcon-fontSizeSmall",
                          "root": "MuiSvgIcon-root",
                        }
                      }
                      style={
                        Object {
                          "fontSize": 200,
                        }
                      }
                    >
                      <svg
                        aria-hidden={true}
                        className="MuiSvgIcon-root"
                        focusable="false"
                        style={
                          Object {
                            "fontSize": 200,
                          }
                        }
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M17.81 4.47c-.08 0-.16-.02-.23-.06C15.66 3.42 14 3 12.01 3c-1.98 0-3.86.47-5.57 1.41-.24.13-.54.04-.68-.2-.13-.24-.04-.55.2-.68C7.82 2.52 9.86 2 12.01 2c2.13 0 3.99.47 6.03 1.52.25.13.34.43.21.67-.09.18-.26.28-.44.28zM3.5 9.72c-.1 0-.2-.03-.29-.09-.23-.16-.28-.47-.12-.7.99-1.4 2.25-2.5 3.75-3.27C9.98 4.04 14 4.03 17.15 5.65c1.5.77 2.76 1.86 3.75 3.25.16.22.11.54-.12.7-.23.16-.54.11-.7-.12-.9-1.26-2.04-2.25-3.39-2.94-2.87-1.47-6.54-1.47-9.4.01-1.36.7-2.5 1.7-3.4 2.96-.08.14-.23.21-.39.21zm6.25 12.07c-.13 0-.26-.05-.35-.15-.87-.87-1.34-1.43-2.01-2.64-.69-1.23-1.05-2.73-1.05-4.34 0-2.97 2.54-5.39 5.66-5.39s5.66 2.42 5.66 5.39c0 .28-.22.5-.5.5s-.5-.22-.5-.5c0-2.42-2.09-4.39-4.66-4.39-2.57 0-4.66 1.97-4.66 4.39 0 1.44.32 2.77.93 3.85.64 1.15 1.08 1.64 1.85 2.42.19.2.19.51 0 .71-.11.1-.24.15-.37.15zm7.17-1.85c-1.19 0-2.24-.3-3.1-.89-1.49-1.01-2.38-2.65-2.38-4.39 0-.28.22-.5.5-.5s.5.22.5.5c0 1.41.72 2.74 1.94 3.56.71.48 1.54.71 2.54.71.24 0 .64-.03 1.04-.1.27-.05.53.13.58.41.05.27-.13.53-.41.58-.57.11-1.07.12-1.21.12zM14.91 22c-.04 0-.09-.01-.13-.02-1.59-.44-2.63-1.03-3.72-2.1-1.4-1.39-2.17-3.24-2.17-5.22 0-1.62 1.38-2.94 3.08-2.94 1.7 0 3.08 1.32 3.08 2.94 0 1.07.93 1.94 2.08 1.94s2.08-.87 2.08-1.94c0-3.77-3.25-6.83-7.25-6.83-2.84 0-5.44 1.58-6.61 4.03-.39.81-.59 1.76-.59 2.8 0 .78.07 2.01.67 3.61.1.26-.03.55-.29.64-.26.1-.55-.04-.64-.29-.49-1.31-.73-2.61-.73-3.96 0-1.2.23-2.29.68-3.24 1.33-2.79 4.28-4.6 7.51-4.6 4.55 0 8.25 3.51 8.25 7.83 0 1.62-1.38 2.94-3.08 2.94s-3.08-1.32-3.08-2.94c0-1.07-.93-1.94-2.08-1.94s-2.08.87-2.08 1.94c0 1.71.66 3.31 1.87 4.51.95.94 1.86 1.46 3.27 1.85.27.07.42.35.35.61-.05.23-.26.38-.47.38z"
                        />
                      </svg>
                    </ForwardRef(SvgIcon)>
                  </WithStyles(ForwardRef(SvgIcon))>
                </ForwardRef(FingerprintIcon)>
              </div>
            </div>
          </Logo>
        </div>
      </Portal>
    </TopBarTitle>
    <div
      className="form-group pl-xs-0 pr-xs-0 mt-0 col-sm-6"
    >
      <label
        data-testid="roomcode-label"
      >
        roomcode
      </label>
      <div
        className="room-code"
        data-testid="roomcode"
      >
        TEST
      </div>
    </div>
    <div
      className="form-group pl-xs-0 pr-xs-0 mt-0 col-sm-6"
    >
      <label
        data-testid="player-list-label"
      >
        player_plural
      </label>
      <PlayerList
        data-testid="player-list"
        players={
          Array [
            Object {
              "name": "test1",
            },
            Object {
              "name": "test2",
            },
            Object {
              "name": "test3",
            },
            Object {
              "name": "test4",
            },
            Object {
              "name": "test5",
            },
            Object {
              "name": "test6",
            },
            Object {
              "name": "test7",
            },
            Object {
              "name": "test8",
            },
            Object {
              "name": "test9",
            },
            Object {
              "name": "test10",
            },
          ]
        }
        xs={12}
      >
        <div
          className=" col-xs-12"
          key="0"
        >
          <PlayerListView
            player={
              Object {
                "name": "test1",
              }
            }
          >
            <BlockLabel>
              <label
                className="block-label"
                style={null}
              >
                test1
              </label>
            </BlockLabel>
          </PlayerListView>
        </div>
        <div
          className=" col-xs-12"
          key="1"
        >
          <PlayerListView
            player={
              Object {
                "name": "test2",
              }
            }
          >
            <BlockLabel>
              <label
                className="block-label"
                style={null}
              >
                test2
              </label>
            </BlockLabel>
          </PlayerListView>
        </div>
        <div
          className=" col-xs-12"
          key="2"
        >
          <PlayerListView
            player={
              Object {
                "name": "test3",
              }
            }
          >
            <BlockLabel>
              <label
                className="block-label"
                style={null}
              >
                test3
              </label>
            </BlockLabel>
          </PlayerListView>
        </div>
        <div
          className=" col-xs-12"
          key="3"
        >
          <PlayerListView
            player={
              Object {
                "name": "test4",
              }
            }
          >
            <BlockLabel>
              <label
                className="block-label"
                style={null}
              >
                test4
              </label>
            </BlockLabel>
          </PlayerListView>
        </div>
        <div
          className=" col-xs-12"
          key="4"
        >
          <PlayerListView
            player={
              Object {
                "name": "test5",
              }
            }
          >
            <BlockLabel>
              <label
                className="block-label"
                style={null}
              >
                test5
              </label>
            </BlockLabel>
          </PlayerListView>
        </div>
        <div
          className=" col-xs-12"
          key="5"
        >
          <PlayerListView
            player={
              Object {
                "name": "test6",
              }
            }
          >
            <BlockLabel>
              <label
                className="block-label"
                style={null}
              >
                test6
              </label>
            </BlockLabel>
          </PlayerListView>
        </div>
        <div
          className=" col-xs-12"
          key="6"
        >
          <PlayerListView
            player={
              Object {
                "name": "test7",
              }
            }
          >
            <BlockLabel>
              <label
                className="block-label"
                style={null}
              >
                test7
              </label>
            </BlockLabel>
          </PlayerListView>
        </div>
        <div
          className=" col-xs-12"
          key="7"
        >
          <PlayerListView
            player={
              Object {
                "name": "test8",
              }
            }
          >
            <BlockLabel>
              <label
                className="block-label"
                style={null}
              >
                test8
              </label>
            </BlockLabel>
          </PlayerListView>
        </div>
        <div
          className=" col-xs-12"
          key="8"
        >
          <PlayerListView
            player={
              Object {
                "name": "test9",
              }
            }
          >
            <BlockLabel>
              <label
                className="block-label"
                style={null}
              >
                test9
              </label>
            </BlockLabel>
          </PlayerListView>
        </div>
        <div
          className=" col-xs-12"
          key="9"
        >
          <PlayerListView
            player={
              Object {
                "name": "test10",
              }
            }
          >
            <BlockLabel>
              <label
                className="block-label"
                style={null}
              >
                test10
              </label>
            </BlockLabel>
          </PlayerListView>
        </div>
      </PlayerList>
    </div>
    <AgreePanel
      data-testid="agree-panel"
      inline={true}
      room={
        Room {
          "agreedPlayers": Array [],
          "episodes": Array [],
          "players": Array [
            Object {
              "name": "test1",
            },
            Object {
              "name": "test2",
            },
            Object {
              "name": "test3",
            },
            Object {
              "name": "test4",
            },
            Object {
              "name": "test5",
            },
            Object {
              "name": "test6",
            },
            Object {
              "name": "test7",
            },
            Object {
              "name": "test8",
            },
            Object {
              "name": "test9",
            },
            Object {
              "name": "test10",
            },
          ],
          "raisedHands": Object {},
          "roomcode": "TEST",
          "state": "lobby",
        }
      }
    >
      <p>
        ready_players
      </p>
      <PlayerList
        inline={true}
        players={Array []}
      />
    </AgreePanel>
  </div>
</LobbyView>
`;
